api/v1beta1/common_types.go

```diff
@@ -237,4 +149,7 @@ type PasswordSelector struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default="SimpleCryptoKEK"
 	SimpleCryptoKEK string `json:"simplecryptokek"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default="P11CryptoLogin"
+	P11CryptoLogin string `json:"p11cryptologin"`
```

Comment:
Nit:  Consider renaming to `PKCS11CryptoLogin` or even better `PKCS11PIN`, since this value corresponds to the PKCS#11 PIN as defined in the PKCS#11 spec.   Same reasoning as above, we should consistently use `PKCS11` everywhere instead of a mix of `PKCS11` and `P11`

Reply:
Done